x/qgb/client/tx.go

```diff
@@ -0,0 +1,56 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/celestiaorg/celestia-app/x/qgb/types"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+)
+
+// GetTxCmd returns the transaction commands for this module
+func GetTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      fmt.Sprintf("%s transactions subcommands", types.ModuleName),
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	cmd.AddCommand(CmdRegisterEVMAddress())
+
+	return cmd
+}
+
+func CmdRegisterEVMAddress() *cobra.Command {
```

Comment:
We can have this command to edit the EVM address for a validator (and also changing its name), but I thought that we were planning on including the message to register an EVM address as part of the create validator tx, then add a rule that a create validator should always have a register EVM address message.